File: seed/java-sdk/exhaustive/local-files/src/main/java/com/snippets/Example44.java
package com.snippets;

import com.fern.sdk.SeedApiClient;
import com.fern.sdk.types.TypesObjectWithUnknownField;
import java.util.HashMap;

public class Example44 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.endpointsObject().endpointsObjectGetAndReturnWithUnknownField(
            TypesObjectWithUnknownField
                .builder()
                .unknown(new 
                    HashMap<String, Object>() {{put("key", "value");
                    }})
                .build()
        );
    }
}

File: seed/swift-sdk/exhaustive/Snippets/Example44.swift
import Foundation
import Api

private func main() async throws {
    let client = ApiClient(
        baseURL: "https://api.fern.com",
        token: "<token>"
    )

    _ = try await client.endpointsObject.endpointsObjectGetAndReturnWithUnknownField(request: TypesObjectWithUnknownField(
        unknown: .object([
            "key": .string("value")
        ])
    ))
}

try await main()
